tools/api-markdown-documenter/package.json

```diff
@@ -112,8 +112,8 @@
 	"packageManager": "pnpm@7.33.6+sha512.90e27fd38047f18583f3342f784cc3f187f4d4caac89ebc1fffae18dcd7b2dde7678a0bf237481bcb8f7e8e66135fa34803856e4eb2c442ce082ffab5d9d241f",
 	"fluidBuild": {
 		"tasks": {
-			"build:tsc": [
-				"^tsc"
+			"tsc": [
+				"^build:esnext"
```

Comment:
Why not rename the script to build:esnext instead of tsc?

Reply:
Because I was not touching scripts and potentially breaking workflows I didn't know about. Leave that for later/others. tools is outside of the fits a standard pattern world and that is not what this task is about.

Reply:
I don't see any references to build:tsc in the repo other than the npm script above and the task entry here... not sure what either of them is even accomplishing since nothing else triggers them through `fluid build`. If we change the task entry I think we want to remove the npm script too. @Josmithr might be able to confirm?

Reply:
We don't need to remove the script entry even if there was a tie in from something else to `build:tsc`. `build:tsc` runs `tsc` which the fluid build system automatically recognizes as a dependency. So, any `build:tsc` task in the system has these manually configured dependencies.
If `build:tsc` script is excessive (unused), then it could be removed.